src/components/App.tsx

```diff
@@ -53,6 +59,31 @@ class App extends React.Component<Record<string, unknown>, AppState> {
       </div>
     );
   }
+
+  private async fetchAave(address, ethPrice) {
```

Comment:
nit: render() is always the bottom most function

Reply:
good point, moved